tests/test_float16.py

```diff
@@ -1,6 +1,8 @@
 import unittest
 import numpy as np
 import onnxruntime as _ort
+import onnxmltools
+import os
```

Comment:
you should move import os, copy before import numpy.

Reply:
> you should move import os, copy before import numpy.

moved and committed.